tests/unittest.jl

```diff
@@ -1,11 +1,16 @@
 include("datamodel/ExampleMC.jl")
+using .ExampleMCModule: ExampleMC
```

Comment:
Is it possible to bring the `ExampleMCCollection` here via `using .ExampleMCModule: ExampleMC, ExampleMCCollection`?

Reply:
simply doing  `include("ExampleMCCollection")` works as of now which also makes the ExampleMC. We can call both ExampleMCCollection( ) and ExampleMC( ) directly.
Since we are bringing the Module out from its namespace anyway in the Collection, we do not explicitly need to include ExampleMC. so L1 and L2 can be removed.